samples/bluetooth/mesh/light_ctrl/CMakeLists.txt

```diff
@@ -15,3 +19,11 @@ target_sources(app PRIVATE
 	src/lc_pwm_led.c)
 target_include_directories(app PRIVATE include)
 # NORDIC SDK APP END
+
+# Preinitialization related to Thingy:53 DFU
+target_sources_ifdef(CONFIG_BOARD_THINGY53_NRF5340_CPUAPP app PRIVATE
```

Comment:
What is the diff between `CONFIG_BOARD_THINGY53_NRF5340_CPUAPP` and `CONFIG_BOARD_THINGY53_NRF5340_CPUAPP_NS` purposes?

Reply:
Im not really sure. It follows the same pattern as in light/light_switch.